mod/wall_attach.php

```diff
@@ -45,32 +45,35 @@ function wall_attach_post(App $a) {
 	$page_owner_nick  = $r[0]['nickname'];
 	$community_page   = (($r[0]['page-flags'] == Contact::PAGE_COMMUNITY) ? true : false);
 
-	if((local_user()) && (local_user() == $page_owner_uid))
+	if ((local_user()) && (local_user() == $page_owner_uid)) {
 		$can_post = true;
-	else {
-		if($community_page && remote_user()) {
+	} else {
+		if ($community_page && remote_user()) {
 			$contact_id = 0;
-			if(is_array($_SESSION['remote'])) {
-				foreach($_SESSION['remote'] as $v) {
-					if($v['uid'] == $page_owner_uid) {
+
+			if (is_array($_SESSION['remote'])) {
+				foreach ($_SESSION['remote'] as $v) {
+					if ($v['uid'] == $page_owner_uid) {
 						$contact_id = $v['cid'];
 						break;
 					}
 				}
 			}
-			if($contact_id) {
 
+			if($contact_id > 0) {
```

Comment:
Standards: Please add a space after `if`.

Reply:
Added a few more in same file.